components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -161,9 +161,9 @@ func UnmanagedArgoWorkFlowExists(ctx context.Context,
 	}
 	// Verify if existing workflow is deployed by ODH with label
 	odhLabelValue, odhLabelExists := workflowCRD.Labels[labels.ODH.Component(ComponentName)]
-	if odhLabelExists && odhLabelValue != "true" {
-		return fmt.Errorf(" %v CRD already exists but not deployed by this operator. Remove existing Argo workflows or set datasciencepipelines to Removed to proceed ",
-			ArgoWorkflowCRD)
+	if odhLabelExists && odhLabelValue == "true" {
+		return nil
```

Comment:
@zdtsw I wonder what `app.opendatahub.io/dsp: false` means and how this would end up being set to such a value?

Reply:
i dont think it will be done by our code base
this is only to prevent if such resource in the cluster with a label done by user themselves.